uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -85,10 +87,24 @@ define([
     };
 
     (function init () {
-      vm.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes);
+      loadSupportedFileTypes();
```

Comment:
This is now async, should not it be loaded via `$q` so we are sure that `vm.supportedFileTypes` are available when calling `vm.request.loadAttachments();` ?